controllers/flowlogspipeline/flp_objects.go

```diff
@@ -252,13 +265,83 @@ func (b *builder) addTransformStages(lastStage *config.PipelineBuilderStage) {
 	}
 
 	// prometheus stage (encode) configuration
+	// the aggregate and prometheus rules will be incorporated from the metrics definitions file
 	agg := enrichedStage.Aggregate("aggregate", []api.AggregateDefinition{})
 	agg.EncodePrometheus("prometheus", api.PromEncode{
 		Port:   int(b.desired.PrometheusPort),
-		Prefix: "flp_",
+		Prefix: "netobserv_",
 	})
 }
 
+var generateStages = []string{"extract_aggregate", "encode_prom"}
+
+func (b *builder) mergeMetricsConfiguration(pipeline config.PipelineBuilderStage) {
+	// set confgen.Opt.SrcFolder, etc
+	confgen.Opt.SrcFolder = b.desired.MetricsDefinitionsDir
```

Comment:
This is going to work when the operator is running locally, but not when it is deployed in kube. There are missing steps to make this directory accessible (e.g. via a configmap / mounted volume).

Without it, the operator crashes, so that's a blocker.

Or another approach could be to use again go embed, but embedding the whole `metrics_definitions` directory, and keep the conf-generation at runtime (it's necessary to apply the `IgnoreMetrics` config).

Reply:
In this case (using embed), the `metrics_definition` directory would not be in the CRD, but in a fixed place. @jotak @eranra Is this agreed?